<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountryAdm</class>
 <widget class="QDialog" name="CountryAdm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>365</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Continente</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="lineEdit_continentCountry"/>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="btn_countryBoard">
     <property name="text">
      <string>Quadro de Pa√≠ses</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="lineEdit_nameCountry"/>
   </item>
   <item row="0" column="3">
    <widget class="QPushButton" name="btn_addCountry">
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="btn_updateCountry">
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="btn_modalities">
     <property name="text">
      <string>Modalidades</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="btn_removeCountry">
     <property name="text">
      <string>Remover</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
